enaml/qt/qt_mpl_canvas.py

```diff
@@ -9,18 +9,26 @@
 
 from enaml.widgets.mpl_canvas import ProxyMPLCanvas
 
-from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg
-try:
-    from matplotlib.backends.backend_qt4agg import (NavigationToolbar2QTAgg
-                                                    as NavigationToolbar2QT)
-except ImportError:
-    from matplotlib.backends.backend_qt4agg import NavigationToolbar2QT
-
-from .QtCore import Qt
+from qtpy import QT_VERSION
+from .QtCore import Qt, PYQT_VERSION_STR
 from .QtWidgets import QFrame, QVBoxLayout
 
 from .qt_control import QtControl
 
+if QT_VERSION[0] == '4':
+    from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg
+    try:
+        from matplotlib.backends.backend_qt4agg import (NavigationToolbar2QTAgg
+                                                        as NavigationToolbar2QT)
+    except ImportError:
+        from matplotlib.backends.backend_qt4agg import NavigationToolbar2QT
+elif QT_VERSION[0] == '5':
+    from matplotlib.backends.backend_qt5agg import FigureCanvasQTAgg
+    from matplotlib.backends.backend_qt5agg import NavigationToolbar2QT
+else:
+    raise RuntimeError('Using qtpy.QT_VERSION == {} which we do not know how '
```

Comment:
Here I find the message a bit confusing. I would say: 
```python
"No known Matplotlib backend for qt version {__version__} (as reported by qtpy.QT_VERSION)"
```
I am not convinced mentioning qtpy here really helps but I am not opposed to keeping the mention